### PR TITLE
Issue #300: Fix drush aliases not scaffolded for nginx hosts

### PR DESCRIPTION
**Pull request by oxyc**

### CODE REVIEW

**Comment by oxyc** [comment]
Also blacklisted the dev tools hosts that are defined in the `example.config.yml`

**Comment by oxyc** [comment]
@geerlingguy you know Ansible inside and out. Are the variables being defined an intended and stable behaviour?

File: provisioning/templates/drupalvm-local.aliases.drushrc.php.j2 [hunk 1]
```diff
@@ -6,10 +6,26 @@
  * @see example.aliases.drushrc.php.
  */
 
-{% for vhost in apache_vhosts %}
-$aliases['{{ vhost.servername }}'] = array(
-  'uri' => '{{ vhost.servername }}',
-  'root' => '{{ vhost.documentroot }}',
+{% set devtool_hosts = [adminer_install_dir, pimpmylog_install_dir, '/usr/share/php/xhprof_html'] %}
```
Comment (geerlingguy) [inline provisioning/templates/drupalvm-local.aliases.drushrc.php.j2]: One note; I think the xhprof path differs on CentOS. I think we might need to add a variable upstream for that, similar to the other install_dir vars you're using.

File: provisioning/templates/drupalvm.aliases.drushrc.php.j2 [hunk 2]
```diff
@@ -6,13 +6,29 @@
...
 
-{% for vhost in apache_vhosts %}
-$aliases['{{ vhost.servername }}'] = array(
-  'uri' => '{{ vhost.servername }}',
-  'root' => '{{ vhost.documentroot }}',
-  'remote-host' => '{{ vhost.servername }}',
-  'remote-user' => 'vagrant',
+{% set devtool_hosts = [adminer_install_dir, pimpmylog_install_dir, '/usr/share/php/xhprof_html'] %}
+
+{% macro alias(host, root) %}
```
Comment (geerlingguy) [inline provisioning/templates/drupalvm.aliases.drushrc.php.j2]: I like this; didn't think of defining a macro here.

**Comment by geerlingguy** [comment]
@oxyc - Can you fix this in line with #410 which was just merged?

**Comment by oxyc** [comment]
Hups

**Comment by frob** [comment]
Way to do @oxyc ++

**Pull request merged**